src/crowdin/index.ts

```diff
@@ -67,7 +67,11 @@ export class CrowdinTranslationSource implements ITranslationSource {
             this.#languageKeys.delete(language.locale);
             this.#languageData.delete(language.locale);
         } else {
-            this.#languageKeys.set(language.locale, { name: language.name, keys: new Set(lookup.keys()) });
+            this.#languageKeys.set(language.locale, {
+                name: lookup.get('config.localeName') ?? language.name,
+                keys: new Set(lookup.keys())
+            });
+            lookup.delete('config.localeName');
```

Comment:
why do you delete it?

Reply:
Just so it doesnt contribute to any stats, this is the only place its used

Reply:
what's the goal of the lookup object actually?

Reply:
It contains the mapping of keys to translated templates, so the english templates can be swapped out